migrations/915-rename-addon-tags.sql

```diff
@@ -0,0 +1 @@
+RENAME TABLE `users_tags_addons` TO `addons_tags`;
```

Comment:
In theory, this does not require a full copy and is supposed to be almost instantaneous, so we should be ok. Maybe we should make a quick test though ? I assume we have lots of data in there.